src/main/java/com/google/devtools/build/lib/profiler/Profiler.java

```diff
@@ -398,7 +399,7 @@ void clear() {
      */
```

Comment:
I don't think there's a point in keeping ProfiledTaskKinds around. How about just turning the command line into an EnumSet (of included tasks)? I think that would simplify the code below.